### ISSUE TITLE
1.17.1 Support

### ISSUE THREAD

**Issue by Alex134831** [TheRandomLabs/RandomPatches#236]
Please add Fabric 1.17.1 support, I'm having an issue: https://www.reddit.com/r/admincraft/comments/op1y3a/friends_cant_play_in_my_locally_hosted_fabric
and i think increasing timeouts like RandomPatches may fix it, but i can't find any other solution for Fabric 1.17.1

**Comment by Alex134831** [comment]
Yes RandomPatches with default config does fix my issue when i try it on 1.16.5!

**Comment by MSDNicrosoft** [comment]
@TheRandomLabs please

**Comment by Boobies** [comment]
Don't be rude. I don't know whether they've lost interest altogether but they are currently working on Scoop and haven't touched the RandomPatches repository in over 4 months. Prodding them annoyingly isn't going to get you anywhere.

**Comment by K0-RR** [comment]
Maybe 1.18?

**Comment by Boobies** [comment]
Was this comment really worth making? You think the dev will go "Ah, that's a great suggestion, how didn't I think of that?"

We get pointless notifications when these content-free replies are made.

**Comment by K0-RR** [comment]
There is no need to be this sarcastic.
There would be no comments like the one I wrote if the dev would have stated they will/won't be updating their mod. I'm also letting them know there is still some interest in the mod.

>We get pointless notifications when these content-free replies are made.

So you thought you will just add another content-free reply?

Edit: And here you are, making another reply, denying yourself - bringing no content again. This time you raise the bar by also being rude just because I just said the truth. Hilarious.

**Comment by Boobies** [comment]
This is a bug tracker, not social media. The content probably went over your head. It has to do with eliminating vermin.
